kindlecomicconverter/KCC_gui.py

```diff
@@ -452,12 +452,23 @@ def selectDir(self):
         if self.needClean:
             self.needClean = False
             GUI.jobList.clear()
+        
         dname = QtWidgets.QFileDialog.getExistingDirectory(MW, 'Select directory', self.lastPath)
         if dname != '':
             if sys.platform.startswith('win'):
                 dname = dname.replace('/', '\\')
             self.lastPath = os.path.abspath(os.path.join(dname, os.pardir))
+            
             GUI.jobList.addItem(dname)
+            
+            for root, _, files in os.walk(dname):
+                for file in files:
+                    if file.lower().endswith('.cbz'):
+                        file_path = os.path.join(root, file)
+                        if sys.platform.startswith('win'):
+                            file_path = file_path.replace('/', '\\')
```

Comment:
I'd do the research, but pretty sure this kind of code is a relic of older Python/Windows and can be removed in modern computers repo-wide. (maybe in a separate PR).

os.path.join handles any conversions.

Reply:
done.